tor/validation/formatting_validation.py

```diff
@@ -75,6 +75,12 @@
 # Image Transcription
 VALID_HEADERS = ["Audio Transcription", "Image Transcription", "Video Transcription"]
 
+# Regex to recognize unescaped reference links.
+# Example:
+#
+# [Test]: Something here
+REFERENCE_LINK_PATTERN = re.compile(r"(?:[^\\]|^)\[.*]:.*")

```

Comment:
Can you escape the closing `]` too? That makes it a bit easier to read, indicating that it's not a character group.

Reply:
Ah, it likely got removed as redundant during a code formatting. Fixed in latest commit.